UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -1935,6 +2026,17 @@ public void SetUpCharacter(PlayerHealthData RaceBodyparts)
 
 		public void InitialiseFromRaceData(PlayerHealthData RaceBodyparts)
 		{
+			foreach (var System in RaceBodyparts.Base.SystemSettings)
+			{
+				continue; //TODO temp
```

Comment:
this means that this doesn't get setup at all, whats the point of this?

Reply:
it's using the old system for now and just having the code for the new system in the game